app/helpers/storage_service_helper/textual_summary.rb

```diff
@@ -0,0 +1,82 @@
+module StorageServiceHelper::TextualSummary
+  include TextualMixins::TextualDescription
+  include TextualMixins::TextualGroupTags
+  include TextualMixins::TextualName
+  include TextualMixins::TextualCustomButtonEvents
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name description version
+      ]
+    )
+  end
+
+  def textual_group_capabilities
+    capabilities = @record.capabilities.map do |c|
+      [c["name"], c["value"]]
+    end
+
+    TextualMultilabel.new(
+      _("Capabilities"),
+      :labels => [_("Capability"), _("Value")],
+      :values => capabilities
+    )
+  end
+
+  def textual_group_storage_resources
+    resources = @record.storage_resources.map do |r|
+      free_space = number_to_percentage(r.logical_free.to_f * 100 / r.logical_total.to_f, :precision => 1)
```

Comment:
Shouldn't we check that `r.logical_total.to_f` isn't `0`?

Reply:
it now looks like this if logical_total is 0:
![image](https://user-images.githubusercontent.com/106743023/217758970-d4b924ec-af19-445f-a2b7-8d5c8670ab61.png)